test/test_ops.py

```diff
@@ -0,0 +1,413 @@
+            AllowedOpInfoEntry('trunc'),
+            AllowedOpInfoEntry('exp2'),
+            AllowedOpInfoEntry('nan_to_num'),
+            AllowedOpInfoEntry('square'),
+            AllowedOpInfoEntry('lerp'),
+            AllowedOpInfoEntry('linalg.inv'),
+            AllowedOpInfoEntry('linalg.inv_ex'),
+            AllowedOpInfoEntry('angle'),
+            AllowedOpInfoEntry('linalg.solve'),
+            AllowedOpInfoEntry('linalg.matrix_rank'),
+            AllowedOpInfoEntry('linalg.matrix_rank', 'hermitian '),
+            AllowedOpInfoEntry('linalg.pinv', 'hermitian '),
+            AllowedOpInfoEntry('einsum'),
+            AllowedOpInfoEntry('linalg.svd'),
+            AllowedOpInfoEntry('linalg.svdvals'),
+            AllowedOpInfoEntry('polar'),
+            AllowedOpInfoEntry('ravel'),
+            AllowedOpInfoEntry('reshape'),
+            AllowedOpInfoEntry('reshape_as'),
+            AllowedOpInfoEntry('view'),
+            AllowedOpInfoEntry('view_as'),
+            AllowedOpInfoEntry('put'),
+            AllowedOpInfoEntry('take'),
+            AllowedOpInfoEntry('stack'),
+            AllowedOpInfoEntry('hstack'),
+            AllowedOpInfoEntry('hypot'),
+            AllowedOpInfoEntry('histogram'),
+            AllowedOpInfoEntry('vstack'),
+            AllowedOpInfoEntry('dstack'),
+            AllowedOpInfoEntry('unfold'),
+            AllowedOpInfoEntry('msort'),
+            AllowedOpInfoEntry('movedim'),
+            AllowedOpInfoEntry('renorm'),
+            AllowedOpInfoEntry('fill_'),
+            AllowedOpInfoEntry('resize_'),
+            AllowedOpInfoEntry('resize_as_'),
+            AllowedOpInfoEntry('take_along_dim'),
+            AllowedOpInfoEntry('unsqueeze'),
+            AllowedOpInfoEntry('xlogy'),
+            AllowedOpInfoEntry('zero_'),
+            AllowedOpInfoEntry('special.xlog1py'),
+            AllowedOpInfoEntry('special.zeta'),
+            AllowedOpInfoEntry('special.zeta', 'grad '),
+            AllowedOpInfoEntry('trace'),
+            AllowedOpInfoEntry('tril'),
+            AllowedOpInfoEntry('triu'),
+            AllowedOpInfoEntry('kron'),
+            AllowedOpInfoEntry('inner'),
+            AllowedOpInfoEntry('tensordot'),
+            AllowedOpInfoEntry('logcumsumexp'),
+            AllowedOpInfoEntry('digamma'),
+            AllowedOpInfoEntry('special.entr'),
+            AllowedOpInfoEntry('special.ndtri'),
+            AllowedOpInfoEntry('lgamma'),
+            AllowedOpInfoEntry('logdet'),
+            AllowedOpInfoEntry('log_softmax'),
+            AllowedOpInfoEntry('log_softmax', 'dtype '),
+            AllowedOpInfoEntry('logit'),
+            AllowedOpInfoEntry('where'),
+            AllowedOpInfoEntry('norm', 'nuc '),
+            AllowedOpInfoEntry('norm', 'fro '),
+            AllowedOpInfoEntry('norm', 'inf '),
+            AllowedOpInfoEntry('special.erfcx'),
+
+            # Duplicate Redundant entries for this test.
```

Comment:
This is awesome, thorough diligence cataloguing these issues. @JackCaoG I'm curious to follow-up in a month or two and see if these were helpful.

Reply:
I am actually not sure what `Duplicate Redundant entries` means here, did the op info test we allowed above already test what these test intended to test?

Reply:
Yes, you are right. We have multiple OpInfo entries (for few Ops) which have more specialized tests in `test_unary_ufuncs`. 
https://github.com/pytorch/pytorch/blob/39ce29efe05dc09fe9de6919715548476e4ab7b6/torch/testing/_internal/common_methods_invocations.py#L7027-L7030